stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -144,6 +144,11 @@ public sealed interface ChatDomain {
      * @return Executable async [Call] responsible for obtainging list of historical [ChatEvent] objects.
      */
     @CheckResult
+    @Deprecated(
+        message = "replayEventsForActiveChannels is deprecated. Use extension function ChatClient::replayEventsForActiveChannels instead",
+        replaceWith = ReplaceWith("ChatClient.replayEventsForActiveChannels()"),
```

Comment:
How about updating the `expression` of `ReplaceWith` explicitly and adding an `import` parameter like the below?
We might prevent breaking client codes when using the `Replace with` on IDE.

```kotlin
replaceWith = ReplaceWith(
            expression = "ChatClient.instance().replayEventsForActiveChannels(cid)",
            imports = arrayOf("io.getstream.chat.android.client.ChatClient")),
```

### replace with
![replace](https://user-images.githubusercontent.com/24237865/141130356-19602ed1-4f8e-4f26-be9e-1d62c062337a.png)

### before
I can see still breaking imports and parameters (cid) after using the `Replace with`.

![before](https://user-images.githubusercontent.com/24237865/141130421-55cb9e90-2629-4323-a16d-03dd33ffc12c.png)

### after
we can resolve this by adding imports and updating the `expression` explicitly with parameters.

![after](https://user-images.githubusercontent.com/24237865/141130522-4e4c5ea5-2119-4bfa-ada7-e3dd1360af30.png)

Reply:
I absolutely agree here. We should have it for all our deprecated functions.